src/js/thirdparty/ws.js

```diff
@@ -67,14 +65,18 @@ class BunWebSocket extends EventEmitter {
         this.emit("message", this.#fragments ? [encoded] : encoded, isBinary);
       }
     });
+    ws.addEventListener("ping", ({ data }) => {
```

Comment:
this should only be added if the user starts listening for ping and pong?

Reply:
It's already somewhat lazy. We don't generate the `ping` or `pong` event unless there is a listener in `WebSocket.cpp`. Should it be lazy here too?